projects/burnr/src/components/NodeSelector.tsx

```diff
@@ -1,193 +1,157 @@
-	};
+  const toggleOpen = () => {
+    setOpen(!open);
+  };
 
-	const open = Boolean(anchorEl);
+  const handleClose = (event?: ChangeEvent<unknown>, reason?: AutocompleteCloseReason) => {
+    if (reason === 'toggleInput') {
+      return;
+    }
+    setOpen(false);
+  };
 
-	return (
-		<div className={classes.wrapper}>
-			<div className={classes.root + (open ? ' node-selector' : '')}>
-				<ButtonBase
-					disableRipple
-					className={classes.button}
-					onClick={handleOpenDropdown}
-				>
-					<NodeSelectorSelected provider={ALL_PROVIDERS[provider]}/>
-					<ArrowDropDownIcon />
-				</ButtonBase>
+  const updateProvider = (provider: string) => {
+    setLocalEndpoint(provider);
+    setProvider(provider);
+    
+    console.log("Burnr wallet is now connected to", ALL_PROVIDERS[provider].endpoint);
+    // Tis is just a temporary work around. Api should be passed on as prop without reload
+    location.reload();
+    // setChain(REMOTE_PROVIDERS[selectedEndpoint].network);
+  };
 
-				{ open &&
-				<>
-					<Typography
-						variant='overline'
-						color='textSecondary'
-						className={classes.acHeader}
-						component='div'
-					>
-						Select node provider
-					</Typography>
+  return (
+    <ClickAwayListener onClickAway={handleClose}>    
+      <div className={classes.nodeSelectorWrap}>
+        <div className={`${classes.nodeSelectorInner} ${open ? 'open' : ''}`}>
 
-					<Autocomplete
-						options={options}
-						disablePortal={true}
-						getOptionLabel={(option) => (option.client === 'string')
-							? `${option.client} client`
-							: `client`
-						}
-						open
-						classes={{
-							popper: classes.acPopper,
-							option: classes.option,
-							paper: classes.acPaper,
-						}}
-						onClose={handleClose}
-						// eslint-disable-next-line @typescript-eslint/no-explicit-any
-						onChange={(event: ChangeEvent<unknown>, {provider: selected}: any ) => {
-							updateProvider(selected)
-						}}
+          <Box display='flex' alignItems='center' pt={1.5} pb={1.5} pl={0.5} pr={0.5} onClick={toggleOpen}>
+            <FiberManualRecordIcon style={{ fontSize: '16px', marginRight: 4 }} color={api && api.isReady ? 'primary' : 'error'} />
+            <Box width='100%'>
+              <Typography variant='h4'>{ ALL_PROVIDERS[provider].network }</Typography>
+              <Typography variant='body2' color='textSecondary'>{ALL_PROVIDERS[provider].client} client</Typography>
+            </Box>
+            {open ? <ArrowDropUpIcon /> : <ArrowDropDownIcon /> }
+          </Box>
 
-						renderInput={(params) => (
-							<InputBase
-								ref={params.InputProps.ref}
-								inputProps={params.inputProps}
-								autoFocus
-								className={classes.acInput}
-							/>
-						)}
-						renderOption={(option) => (
-							<NodeSelectorItem provider={option} selected={option.provider === provider} />
-						)}
-						groupBy={(option) => option.network}
-					/>
-				</>
-				}
+          {open &&
```

Comment:
Same as previous comment here: it should be:
```suggestion
          {open ?
```